index.php

```diff
@@ -1 +1,44 @@
+<html lang="en">
+<head>
+    <title>Crep Collector</title>
+    <link rel="stylesheet" href="style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com" rel="stylesheet">
```

Comment:
I'm not sure that this is actually going to load a custom font stylesheet - the URL doesn't look long enough to me. Do the fonts work?